cypress/support/config/services.js

```diff
@@ -388,7 +388,7 @@ module.exports = {
             : '/igbo/articles/cxvxrj8tvppo',
         smoke: true,
       },
-      frontPage: { path: '/igbo', smoke: true },
+      frontPage: { path: '/igbo', smoke: false },
```

Comment:
🤔 Why are we turning smoke off for `igbo`?

Reply:
Because we're turning it on for Thai :) It's already off for Yoruba and Pidgin, we only need the most complex front page smoke tested.